support-frontend/assets/pages/[countryGroupId]/components/createSubscription.ts

```diff
@@ -0,0 +1,131 @@
+import type { ErrorReason } from 'helpers/forms/errorReasons';
```

Comment:
👍  Thanks for doing this. I think it's nice to see all of this functionality grouped into one file.